crates/sui-axelar-cgp/move/sources/utils.move

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+module axelar::utils {
+    use std::vector;
+
+    use sui::bcs;
+    use sui::hash;
+
+    /// Normalize last byte of the signature. Have it 1 or 0.
+    /// See https://tech.mystenlabs.com/cryptography-in-sui-cross-chain-signature-verification/
+    public fun normalize_signature(signature: &mut vector<u8>) {
+        // Compute v = 0 or 1.
+        let v = vector::borrow_mut(signature, 64);
+        if (*v == 27) {
+            *v = 0;
+        } else if (*v == 28) {
+            *v = 1;
+        } else if (*v > 35) {
+            *v = (*v - 1) % 2;
+        };
+    }
+
+    /// Prefix for Sui Messages.
+    const PREFIX: vector<u8> = b"\x19Sui Signed Message:\n";
```

Comment:
Constants are usually placed at the top although I see the rationale for making it closer.